examples/hooks/App.tsx

```diff
@@ -61,119 +62,131 @@ export function App() {
+                items={[
+                  { label: '20 hits per page', value: 20, default: true },
+                  { label: '40 hits per page', value: 40 },
                 ]}
-                showMore={true}
               />
-            </Panel>
-            <Panel header="Price">
-              <RangeInput attribute="price" />
-            </Panel>
-            <Panel header="Free Shipping">
-              <ToggleRefinement
-                attribute="free_shipping"
-                label="Free shipping"
+              <SortBy
+                items={[
+                  { label: 'Relevance', value: 'instant_search' },
+                  { label: 'Price (asc)', value: 'instant_search_price_asc' },
+                  { label: 'Price (desc)', value: 'instant_search_price_desc' },
+                ]}
               />
-            </Panel>
-          </DynamicWidgets>
-        </div>
-        <div className="Search">
-          <Breadcrumb
-            attributes={[
-              'hierarchicalCategories.lvl0',
-              'hierarchicalCategories.lvl1',
-              'hierarchicalCategories.lvl2',
-            ]}
-          />
+              <Refresh />
+            </div>
 
-          <SearchBox placeholder="Search" autoFocus />
+            <div className="CurrentRefinements">
+              <ClearRefinements />
+              <CurrentRefinements
+                transformItems={(items) =>
+                  items.map((item) => {
+                    const label = item.label.startsWith(
+                      'hierarchicalCategories'
+                    )
+                      ? 'Hierarchy'
+                      : item.label;
 
-          <div className="Search-header">
-            <PoweredBy />
-            <HitsPerPage
-              items={[
-                { label: '20 hits per page', value: 20, default: true },
-                { label: '40 hits per page', value: 40 },
-              ]}
-            />
-            <SortBy
-              items={[
-                { label: 'Relevance', value: 'instant_search' },
-                { label: 'Price (asc)', value: 'instant_search_price_asc' },
-                { label: 'Price (desc)', value: 'instant_search_price_desc' },
-              ]}
-            />
-            <Refresh />
-          </div>
+                    return {
+                      ...item,
+                      attribute: label,
+                    };
+                  })
+                }
+              />
+            </div>
 
-          <div className="CurrentRefinements">
-            <ClearRefinements />
-            <CurrentRefinements
-              transformItems={(items) =>
-                items.map((item) => {
-                  const label = item.label.startsWith('hierarchicalCategories')
-                    ? 'Hierarchy'
-                    : item.label;
+            <Index
+              indexName="autocomplete_demo_products_query_suggestions"
+              parentIndexId="instant_search"
```

Comment:
more ideally `parentIndexId={null}` would behave as another mainIndex, but not sure how that can be done. It could of course be implemented later as well

Reply:
I think we can resolve `null` to `useInstantSearchContext().indexName`.

Reply:
yep, null could be mainIndex, but it wouldn't really solve the issue here (needing to have a manual index at the root with different indexId), as that mainIndex would probably have widgets as well, negating the point of parentId a bit (I could almost only imagine use cases where you want _no_ parameters to be inherited)

Reply:
I wonder how easier the mental model would be if we require an `<Index>` within `<InstantSearch>`.

Because we can't really portal an `<Index>` to an empty Helper in the current state. The root is shared among all `<Index>`.

(I see this problem a bit like Element and Fragment.)

Reply:
Yep, that's what I had in mind as well, possibly that could be detected when the InstantSearch instance doesn't receive an indexName prop, you render Index manually, and thus will be able to have adjacent root indices